pkg/kudoctl/cmd/version.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"fmt"
+
```

Comment:
can we collapse these lines?

Reply:
Personal preference here too:. I like having the K8s packages in their own region as well